driver/tke_driver.go

```diff
@@ -865,32 +946,39 @@ func (d *Driver) GetVersion(ctx context.Context, info *types.ClusterInfo) (*type
 	if err != nil {
 		return nil, err
 	}
-	return &types.KubernetesVersion{Version: resp.Data.Clusters[0].K8sVersion}, nil
+	return &types.KubernetesVersion{Version: *resp.Response.Clusters[0].ClusterVersion}, nil
 }
 
 // operateClusterVip creates or remove the cluster vip
 func operateClusterVip(ctx context.Context, svc *ccs.Client, clusterID, operation string) error {
 	logrus.Info("invoking operateClusterVip")
 
-	req := ccs.NewOperateClusterVipRequest()
-	req.ClusterID = clusterID
-	req.Operation = operation
+	req := ccs.NewCreateClusterEndpointVipRequest()
+	req.ClusterID = &clusterID
+
+	reqStatus := ccs.NewDescribeClusterEndpointVipStatusRequest()
+	reqStatus.ClusterID = &clusterID
+
+	_, err := svc.CreateClusterEndpointVip(req)
+
+	if err != nil {
```

Comment:
Use this common pattern: `if  _, err := svc.CreateClusterEndpointVip(req);err!=nil{}`